src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -0,0 +1,184 @@
+"use strict";
```

Comment:
`use strict` is not needed here, because it will be automatically added in the build process.